docs/schemas/Cell-Annotation-and-Taxonomy/Data and Metadata/Metadata/Assigned Metadata/Taxonomy_AssignedMetadata.README.md

```diff
@@ -0,0 +1,73 @@
+# Assigned Metadata Schema
+
+Document Status: _Under MOWG Review_
+
+Version: 1.0
+
+Owner: @UCDNJJ, @jeremymiller
+
+Reviewers: @patrick-lloyd-ray, @carolth, @djarecka, @memartone
+
+License: [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Date Created: 10-03-2025
+
+## Overview
+
+The BICAN Taxonomy Assigned Metadata schema specifies the metadata relating to taxonomy metadata that can be taken from a variety of sources in BICAN. These metadata reflect the metadata needed to accurately track taxonomy metadata in BICAN. As such, it is a collaborative schema that reflects the joint efforts of members of BICAN.
+
+There are many metadata schemas that can be used in leiu of those specified in this document. 
+
+* [Developing-Human-Metadata]
+* [Developing-NHP-Metadata]
+* [Developing-Tissue-Metadata]
+* [Donor-Metadata]
+* [Macaque-Donor-and-Tissue-Metadata]
+* [Marmoset-Metadata]
+
+to name a few. If you are not using one of these metadata schemas, feel free to enter your own metadata in the format specified by this document below.
+
+This document has the following sections:
+
+* [General Requirements](#general-requirements)
+* [obs](#obs)
+* [uns](#uns)
+* [changelog](#changelog)
+
+### General Requirements
+
+This includes cell-level metadata that is assigned at some point in the process between when a cell goes from the donor to a value in the data, and (in theory) can be ENTIRELY captured by values in Allen Institute, BICAN, or related standardized pipelines. It includes things like donor metadata, experimental protocols, dissection information, RNA QC metrics, and sequencing metadata. Ideally a schema for this will be defined through other BICAN groups, and can be adopted here.
+
+### obs
+
```

Comment:
this part should have extra empty lines, or other formatting, now it is everything in one paragraph